### DISCUSSION TITLE
Automations Stuck

### DISCUSSION THREAD

**Discussion by aloreaux** [Budibase/budibase#6747] [Help]
Hello I was playing around with automations and it got stuck switching between value and binding (set it to binding, then went back to value and it completely locked up). I now cannot click on the automate link or my app freezes up. I am unable to remove the automation, anyway to fix this? I tried looking through the bbapps docker image but couldn't find anything.

**Comment by melohagan** [comment]
Hi @aloreaux 

Do you see any errors in the console or network tabs? Fn+F12 on Chrome

**Reply by aloreaux** [reply]
Yes this is what I see.

`Uncaught (in promise) TypeError: B.forEach is not a function
    at I (index.91ff43ed.js:23:44930)
    at Object.CL.r.$$.update (index.91ff43ed.js:23:45609)
    at xs (vendor.7f11b0ca.js:4:7804)
    at new SL (index.91ff43ed.js:23:45828)
    at Array.AL (index.91ff43ed.js:23:46007)
    at M9 (vendor.7f11b0ca.js:1:1000725)
    at tE (index.91ff43ed.js:1:11134)
    at xs (vendor.7f11b0ca.js:4:7851)
    at new Gr (index.91ff43ed.js:1:12237)
    at OL (index.91ff43ed.js:23:46607)`

**Reply by melohagan** [reply]
And which version of Budibase and app version are you on?

**Reply by aloreaux** [reply]
Budibase 1.1.4 and app 1.1.2 (although I believe both were 1.1.2 when this error occurred)

**Reply by melohagan** [reply]
Would you be able to provide reproduction steps? Or at least narrow it down to the types of automation steps, bindings, and any other information that could be useful to investigate this please.

**Reply by aloreaux** [reply]
No problem.

1. On row created
2. Conditional on column value
3. Update row (set it to binding, tested and did not work. Changed it to value to continue testing and it locked up)

**Reply by aloreaux** [reply]
So I did export my app, then removed the automation section and it does not freeze anymore. Still doesn't answer why this happens but it is a work around.
